src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+// body {
+// margin: 0;
+// padding: 0;
+
+// display: flex;
+// justify-content: center;
+// font-family: Roboto, sans-serif;
+// }
+
+.product-card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: $card-border-width solid #f3f3f3;
+  padding: $card-padding;
+
+  transition: transform 300ms;
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
The hover effect on the product card scales it to 1.2 times its size. Ensure this does not cause layout issues or overflow in the design.